Backoffice/EventSubscriber/ContentUpdateCacheSubscriber.php

```diff
@@ -34,12 +34,9 @@ public function invalidateCacheOnStatusChanged(ContentEvent $event)
     {
         $content = $event->getContent();
         $previousStatus = $event->getPreviousStatus();
-        $status = $content->getStatus();
 
         if ($previousStatus instanceof StatusInterface &&
-            $status instanceof StatusInterface &&
-            $previousStatus->isPublished() &&
-            !$status->isPublished()) {
+            $previousStatus->isPublished()) {
```

Comment:
@itkg-nanne : I think the correct indent is 
``if ($previousStatus instanceof StatusInterface
    && $previousStatus->isPublished()
) {
``